aws_lambda_powertools/utilities/parameters/appconfig.py

```diff
@@ -100,16 +100,23 @@ def _get(self, name: str, **sdk_options) -> str:
         name: str
             Name of the configuration
         sdk_options: dict, optional
-            Dictionary of options that will be passed to the client's get_configuration API call
+            Dictionary of options that will be passed to the client's start_configuration_session API call
         """
+        if not self.next_call:
+            sdk_options["ConfigurationProfileIdentifier"] = name
+            sdk_options["ApplicationIdentifier"] = self.application
+            sdk_options["EnvironmentIdentifier"] = self.environment
+            response_configuration = self.client.start_configuration_session(**sdk_options)
+            self.next_call = response_configuration["InitialConfigurationToken"]
 
-        sdk_options["Configuration"] = name
-        sdk_options["Application"] = self.application
-        sdk_options["Environment"] = self.environment
-        sdk_options["ClientId"] = CLIENT_ID
+        response = self.client.get_latest_configuration(ConfigurationToken=self.next_call)
```

Comment:
arent you supposed to add here the sdk_options? let's say i want to config from a different region or some other config

Reply:
Thanks for pointing out this @ran-isenberg. Configuration like region, retries, credentials and others are passed to the constructor and not to the API calls, that is, if you check [this line](https://github.com/awslabs/aws-lambda-powertools-python/pull/1553/files#diff-5576360015200fe47fe1113f8fcee3d78e9fdb6059051e7663d220c7dadcf78bR82) you see any additional configuration is passed to the constructor.

I created this fictional code to make sure we are not missing anything
![image](https://user-images.githubusercontent.com/4295173/193814001-f9c698e9-f406-4d2a-a290-fd9517e20cd8.png)
![image](https://user-images.githubusercontent.com/4295173/193814746-4fa042e6-d07d-4c48-bf5e-4950285de7c8.png)
![image](https://user-images.githubusercontent.com/4295173/193814439-349203e7-8f9a-44a1-ba73-f767fc8327cb.png)

I think you mentioned a situation where a customer needs to pass [additional parameters to the SDK](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/parameters/#passing-additional-sdk-arguments) and it makes sense, but I don't think that's the situation here. The method [get_latest_configuration](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/appconfigdata.html#AppConfigData.Client.get_latest_configuration) only supports the `ConfigurationToken` argument and if in the future it supports more than this argument, we need to review the impact of a possible change.

Reply:
yes, my bad! I looked at the wrong api doc. sorry about that.